PlayCover/AppInstaller/Downloader.swift

```diff
@@ -39,6 +39,21 @@ class DownloadApp {
         if installVM.inProgress {
             Log.shared.error(PlayCoverError.waitInstallation)
         } else {
+            if let app = app, PlayApp.PROHIBITED_APPS.contains(app.bundleID) {
+                let alert = NSAlert()
+                alert.messageText = NSLocalizedString("alert.error", comment: "")
+                alert.informativeText = String(
+                    format: NSLocalizedString("error.appProhibited", comment: ""),
+                    arguments: [app.name]
+                )
+                alert.alertStyle = .warning
+                alert.addButton(withTitle: NSLocalizedString("Ok", comment: ""))
+                alert.addButton(withTitle: NSLocalizedString("alert.download.downloadAnyway", comment: ""))
```

Comment:
There is no corresponding string for "alert.download.downloadAnyway" in the localizations file.